cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -441,3 +443,391 @@ def test_conversion_host_tags(appliance, v2v_providers):
+            #     view.progress_card.total_vm_to_be_migrated(migration_plan.name))
+            logger.info("For plan %s, is plan in progress: %s, time elapsed for migration: %s",
+                migration_plan.name, is_plan_visible,
+                view.progress_card.get_clock(migration_plan.name))
+        # return False if plan visible under "In Progress Plans"
+        return not is_plan_visible
+
+    # wait until plan is in progress
+    wait_for(func=_get_plan_status, message="migration plan is in progress, be patient please",
+     delay=5, num_sec=3600)
+    view.migr_dropdown.item_select("Completed Plans")
+    view.wait_displayed()
+    logger.info("For plan %s, migration status after completion: %s, total time elapsed: %s",
+        migration_plan.name, view.migration_plans_completed_list.get_vm_count_in_plan(
+            migration_plan.name), view.migration_plans_completed_list.get_clock(
+            migration_plan.name))
+    assert view.migration_plans_completed_list.is_plan_succeeded(migration_plan.name)
+    # check power state on migrated VM
+    rhv_prov = v2v_providers[1]
+    migrated_vm = rhv_prov.mgmt.get_vm((form_data_vm_obj_single_datastore[1][0]).name)
+    assert power_state in migrated_vm.state
+    # check tags
+    vm_obj = appliance.collections.infra_vms.instantiate(migrated_vm.name, rhv_prov)
+    owner_tag = None
+    for t in vm_obj.get_tags():
+        if tag.display_name in t.display_name:
+            owner_tag = t
+    assert owner_tag is not None and tag.display_name in owner_tag.display_name
+    assert 'Never' not in vm_obj.retirement_date
+
+
+@pytest.mark.parametrize('host_creds, form_data_multiple_vm_obj_single_datastore', [['multi-host',
+    ['nfs', 'nfs', [rhel7_minimal, ubuntu16_template, rhel69_template, win7_template]]]],
+    indirect=True)
+def test_multi_host_multi_vm_migration(request, appliance, v2v_providers, host_creds,
+                                    conversion_tags, soft_assert,
+                                    form_data_multiple_vm_obj_single_datastore):
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+
+    mapping = infrastructure_mapping_collection.create(
+        form_data_multiple_vm_obj_single_datastore[0])
+
+    @request.addfinalizer
+    def _cleanup():
+        infrastructure_mapping_collection.delete(mapping)
+
+    migration_plan_collection = appliance.collections.v2v_plans
+
+    migration_plan = migration_plan_collection.create(
+        name="plan_{}".format(fauxfactory.gen_alphanumeric()), description="desc_{}"
+        .format(fauxfactory.gen_alphanumeric()), infra_map=mapping.name,
+        vm_list=form_data_multiple_vm_obj_single_datastore[1], start_migration=True)
+    # as migration is started, try to track progress using migration plan request details page
+    view = appliance.browser.create_view(navigator.get_class(migration_plan_collection, 'All').VIEW)
```

Comment:
@sshveta wouldn't it be better to keep using current method, in case we change the class name for actual view, the navigation destination will know which class to point to, removing need to update this test. I think we should keep it as is.